commcare_export/commcare_minilinq.py

```diff
@@ -177,7 +177,8 @@ class SimplePaginator(object):
     Paginate based on the 'next' URL provided in the API response.
     """
 
-    def __init__(self, page_size=1000, params=None):
+    def __init__(self, page_size=None, params=None):
+        page_size = page_size if page_size else 1000
```

Comment:
Nit: Could we put 1000 into a const variable? Perhaps something like `DEFAULT_PAGE_SIZE`

Reply:
https://github.com/dimagi/commcare-export/pull/225/commits/a32d003bcf74eda938b8aa869801870b0b2685b5